test/metabase/api/embed_test.clj

```diff
@@ -684,11 +684,8 @@
                      response))))
           (testing "card based param"
             (let [response (dropdown card (:card param-keys))]
-              (is (= {:values          ["Red Medicine"
-                                        "Stout Burgers & Beers"
-                                        "The Apple Pan"
-                                        "Wurstküche"
-                                        "Brite Spot Family Restaurant"]
+              (is (= {:values          ["Brite Spot Family Restaurant" "Red Medicine"
```

Comment:
the result of these tests changed because we're getting distinct values and it has an implicit sort.